README.md

```diff
@@ -90,6 +90,12 @@ Install this project with:
 - GitHub: [@coderacheal](https://github.com/coderacheal)
 - LinkedIn: [racheal-appiah-kubi](https://www.linkedin.com/racheal-appiah-kubi/)
```

Comment:
- [ ] Dear. Please remember we are not expected to have any broken links in our readme file. Please fix the Ruby broken  link ⬇️ 

![image](https://user-images.githubusercontent.com/79058364/235262693-a7fa5a23-fa4f-4a2c-ae81-f9c52c7c6672.png)